src/testrunner/CMakeLists.txt

```diff
@@ -32,6 +32,11 @@ source_group("Header Files" FILES ${folder_header})
 add_executable(${PROJECT_NAME} ${folder_source} ${folder_header})
 target_link_libraries(${PROJECT_NAME} ${RTF_LIBS} ${RTF_INTERNAL_LIBS} ${TinyXML_LIBRARIES})
 
+target_compile_features(${PROJECT_NAME} PUBLIC cxx_nullptr)
+
+target_include_directories(${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
```

Comment:
Not a big problem, but it is a bit strange to have `PUBLIC` include directories/compile features for an executable.

Reply:
Fixed in https://github.com/robotology/robot-testing/pull/82/commits/b85b8dad5acf4cd8c4ac4b7139e7c53add900281 .